jest.config.js

```diff
@@ -2,3 +2,4 @@
 module.exports = {
   verbose: true,
 };
+// collectCoverageFrom: [     '**/*.{js,jsx}',     '!**/public/**',     '!**/coverage/**',     '!**/node_modules/**',     '!**/*.//config.js',   ],
```

Comment:
I copied Joshes from the test repo and went through it. Loads in there most of which we don’t need but useful